src/Kubernetes.WebSocket.cs

```diff
@@ -67,16 +72,24 @@ public Task<WebSocket> WebSocketNamespacedPodExecAsync(string name, string @name
 
             uriBuilder.Path += $"api/v1/namespaces/{@namespace}/pods/{name}/exec";
 
+            List<string> queryParameters = new List<string>();
```

Comment:
I assume you want multi commands param, but I suggest using `queryhelper`

queryhelper cloud help you add multiple params
```
    var q = QueryHelpers.AddQueryString(string.Empty, new Dictionary<string, string>
            {
                {"a", "b"}
            });

            q = QueryHelpers.AddQueryString(q, "command", "1");
            q = QueryHelpers.AddQueryString(q, "command", "1");
            q = QueryHelpers.AddQueryString(q, "command", "2");
            Console.WriteLine(q); // ?a=b&command=1&command=1&command=2
```